src/auth/Auth.cpp

```diff
@@ -102,7 +103,7 @@ namespace SDDM {
         static std::unique_ptr<Auth::SocketServer> self;
         if (!self) {
             self.reset(new SocketServer());
-            self->listen(QStringLiteral("sddm-auth-%1").arg(QUuid::createUuid().toString().replace(QRegExp(QStringLiteral("[{}]")), QString())));
+            self->listen(QStringLiteral("sddm-auth-%1").arg(QUuid::createUuid().toString().replace(QRegularExpression(QStringLiteral("[{}]")), QString())));
```

Comment:
Not a straight port, but this can be replaced by `QUuid::createUuid().toString(QUuid::WithoutBraces)`

Reply:
Yeah, missed it, good idea!